pkg/app/piped/notifier/slack.go

```diff
@@ -111,6 +131,18 @@ func (s *slack) sendEvent(ctx context.Context, event model.NotificationEvent) {
 		return
 	}
 	if len(s.config.OAuthToken) != 0 {
```

Comment:
```suggestion
	if len(s.config.OAuthToken) != 0 || len(s.config.OAuthTokenData) != 0 || len(s.config.OAuthTokenFile) != 0 {
		if err := s.sendMessageViaAPI(ctx, msg); err != nil {
			s.logger.Error(fmt.Sprintf("unable to send notification to slack: %v", err))
		}
		return
	}
```
I guess this would be easier to read

Reply:
you're right :)